### DISCUSSION TITLE
Using content tabs in links.

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [squidfunk/mkdocs-material#3833] [Q&A] [answered]
Is it possible to reference a content tab in a link `[Take me to a content tab](<some magic here>)` which when clicked focuses the content tab ?

**Comment by squidfunk** [comment, marked as answer]
Nope, not possible. Content tabs are implemented with radio buttons. In order to switch to them on link click I'm pretty convinced JavaScript is mandatory, which might break the logic on slow connections. I'm not sure I want to support something that only works half-way.

**Reply by squidfunk** [reply]
We'll re-evaluate this in #3976. I may have an idea how this is fixable.

**Reply by squidfunk** [reply]
This has been implemented, see https://github.com/squidfunk/mkdocs-material/issues/3976#issuecomment-1146586469

**Reply by UltraInstinct05** [reply]
Thanks for implementing it ! One thing I noticed is that when you click on a link (In this case `jump to third tab in paragraph`), it takes you to the third tab (In this case, `... or even me`), but after that, if you click on the first tab (In this case, `Open me in a new tab ...`) and then try to click the link again, it doesn't switch focus back to the 3rd tab (presumably because the browser link already thinks that I am on the third tab, when I am not).

Is this intended ?

![image](https://user-images.githubusercontent.com/34304644/172003468-cc1178de-1cb3-4f29-b2c6-81aeb9598cfa.png)

**Reply by squidfunk** [reply]
It's because the browser still has the target of the third tab in the URL. I wouldn't consider this intended, could you create an issue?

**Reply by UltraInstinct05** [reply]
Done. https://github.com/squidfunk/mkdocs-material/issues/3988 
Note that I have not attached a minimal reproduction zip file because it can be easily reproduced on the website. But the exact steps are given.
